src/sql/workbench/services/notebook/common/sqlSessionManager.ts

```diff
@@ -470,4 +512,16 @@ export interface IDataResourceFields {
 export interface IDataResourceSchema {
 	name: string;
 	type?: string;
-}
\ No newline at end of file
+}
+
+class ExternalScriptMagic {
+
+	constructor(private language: string) {
+	}
+	public convertToExternalScript(script: string): string {
+		return `execute sp_execute_external_script
+		@language = N'${this.language}',
```

Comment:
whenever passing in parameters to a sql query just always concerned about SQL Injection. calling that out here 😃

Reply:
Created #4097